### DISCUSSION TITLE
Make text colors changeable

### DISCUSSION THREAD

**Discussion by jenspfahl-tgtg** [testingisdocumenting/znai#1111] [Ideas]
It would be great to provide an option to alter the used colors for texts, from headers down to the plain text.  This could be configured in the meta.json. 
It woul also be great to have an option to replace the used standard font with a custom provided one. With that the generated guide could be complied to company brand compliances.

**Comment by MykolaGolubyev** [comment]
Hi @jenspfahl-tgtg ,
this should already be possible. Try adding `style.css` file next to `toc` file and put all the css overrides there (consider using css var overrides). 
Alternatively there is an `extensions.json` file (https://github.com/testingisdocumenting/znai/blob/master/znai-docs/znai/extensions.json) you can specify what css/js/html files you want to add as extra.

**Reply by jenspfahl-tgtg** [reply]
Thank you to point me to this solution. Is there already a documentation about that and what else could be customized? Haven't found it. So i am also looking into a way to replace / overwrite the css fonts declarations.

**Reply by MykolaGolubyev** [reply]
No documentation at the moment. Was waiting for users to ask questions to figure out what parts are important to document in this space. 
Now is a good time to start :) ! 
let me know what kind of challenges you are facing. 
My recommendation is to try to stick to changing bar names that start with “znai-“. They are not going to change their name. Same can’t be said about arbitrary css class names. You can check what variables are available using chrome Dev tools.
